### ISSUE TITLE
using md5 hash to place markdown attachment links appropriately

### ISSUE THREAD

**Issue by Analect** [exomut/evernote-dump#3]
@exomut 
I happened upon your very useful tool, which I realise is a work-in-progress .. so thanks.

I've tried it against various `enex` files and notice that, in my case at least, the order of the base64 blobs within `resources` are not necessarily in the order in which files (mostly images, in my case) appear in the body.  I suspect that might be because I go back and add additional items to evernote and don't necessarily 'build' the document in an ordered way.

Anyway, it seems that in order to get an accurate ordering, it might be necessary, after processing each data resource, to calculate an `md5 hash` and isolate which file (with the equivalent hash reference in the body) it references and then replace the file reference at that point to a new file pointer.  [Here](https://github.com/evernote/evernote-sdk-python/blob/master/sample/client/EDAMTest.py#L65-L72) is an illustration of how evernote appear to be generating those hashes.

**Comment by exomut** [comment]
I was hoping I wasn't going to have to use the hash tag. For all of my enex files the output was in the correct order. The link that you provided was great! I was able to implement the hash code with out problems...I think.

I am doing this project in my free time and I recently moved off of Evernote. I had a ton of notes. All of the other export tools would add the export date as the date.

I am slowly adding new features. I have the ideas, just not the time. So check back in a couple of months.
